tests/datasets/test_advance.py

```diff
@@ -57,6 +56,7 @@ def mocked_import(name: str, *args: Any, **kwargs: Any) -> Any:
         )
 
     def test_getitem(self, dataset: ADVANCE) -> None:
+        pytest.importorskip("scipy", minversion="0.9.0")
```

Comment:
Scipy is only needed for `__getitem__`, it isn't needed for any other part of the dataset.